Workspace.swift

```diff
@@ -1,6 +1,42 @@
 import WorkspaceConfiguration
 
 let configuration:WorkspaceConfiguration = .init()
+
+// ••••••• General •••••••
+
+// The project appears to only target Linux.
+// In any case, it fails to compile on macOS right now.
+configuration.supportedOperatingSystems.remove(.macOS)
+configuration.supportedOperatingSystems.remove(.iOS)
+configuration.supportedOperatingSystems.remove(.watchOS)
+configuration.supportedOperatingSystems.remove(.tvOS)
+
+// ••••••• Proofreading •••••••
+
+// These are documented at https://sdggiesbrecht.github.io/Workspace/🇨🇦EN/Types/ProofreadingRule.html
+
 configuration.proofreading.rules.remove(.unicode)
 configuration.proofreading.rules.remove(.colonSpacing)
+
+// Disabled because there are violations.
+// For an explanation, see: https://forums.swift.org/t/introducing-documentation-generation-for-swift-packages/15971/8?u=sdggiesbrecht
+configuration.proofreading.rules.remove(.compatibilityCharacters)
```

Comment:
Is it possible to whitelist *only* the `…` character?

Reply:
Not yet. You can request that feature if you like. (But there are a lot more important things that need fixing first...)

Reply:
This was answered, right?